test/run_test.py

```diff
@@ -149,6 +151,67 @@ def working_directory():
     return os.getcwd()
 
 
+async def read_stdout(stdout, num_lines):
```

Comment:
These two functions could be merged in one with stdout/stderr given by parameter?